.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -10,12 +10,9 @@ assignees: ''
 **Is your feature request related to a problem? Please describe.**
 What's the problem? For example, "I'm always frustrated when..."
 
-Please note that Node.js is not supported by Connect-Web. We are working on
-Connect for Node.js - if you are interested, let us know on Slack or on
-GitHub discussions.  
-
 **Describe the solution you'd like**
-What would you like to have `connect-web` do? How should we solve the problem?
+What would you like to have `connect-web`/`connect-node` do? How should we solve
```

Comment:
I would change this to `connect-es` maybe.  In addition, I'd maybe add a question or statement somewhere like 'Please specify whether this is for Connect for Web or Connect for Node.js.

Reply:
Done.